packages/lib-react-components/src/AnimatedNumber/AnimatedNumber.js

```diff
@@ -75,7 +78,7 @@ function AnimatedNumber({ duration = 1000, value }) {
     return () => {
       intersectionObserver.disconnect()
     }
-  }, [numRef.current, animated])
+  }, [numRef.current, animated, value, duration])
```

Comment:
React refs aren’t reactive values, so I don’t think `numRef` needs to be declared as a dependency here.